### PR TITLE
-Add filtering by activity type; activities not matching the desired …

### PR DESCRIPTION
**Pull request by GregChung-IBFL**
…types will not be processed.

e.g. the command line arg "-workout_types cycling,strength" will process those activities, but skip over any running, stretching, yoga, etc.
-Output to the log basic info about each workout (title, type, timestamp)

### CODE REVIEW

File: lib/pelotonApi.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import requests, json
 import logging
 import sys
+from datetime import datetime
```
Comment (philosowaffle) [inline lib/pelotonApi.py:4]: Don't think we need this import, can you clean it up?

File: lib/configuration.py [hunk 2]
```diff
@@ -32,6 +32,7 @@ def __init__(self, args):
         args.add_argument("-garmin_enable_upload",help="True will try to upload activities to Garmin", dest="garmin_enable_upload", type=str, default=os.environ.get('P2G_GARMIN_ENABLE_UPLOAD'))
         args.add_argument("-path",help="Path to output directory",dest="output_dir",type=str, default=os.environ.get('P2G_PATH'))
         args.add_argument("-num",help="Number of activities to download",dest="num_to_download",type=int, default=os.environ.get('P2G_NUM'))
+        args.add_argument("-workout_types",help="Process specified activity types only", dest="workout_types", type=str, default=os.environ.get('P2G_WORKOUT_TYPES'))
```
Comment (philosowaffle) [inline lib/configuration.py:35]: Great idea for a feature, can you update the readme [Config ](https://github.com/philosowaffle/peloton-to-garmin#configuration) table aswell?

**Review by philosowaffle — CHANGES_REQUESTED** [review]
Thanks for the new feature! Just a few minor feedbacks.

File: lib/configuration.py (hunk 2 above)
Comment (GregChung-IBFL) [reply lib/configuration.py:35]: "Done and dusted."

**Review by philosowaffle — APPROVED** [review]

**Pull request merged**